src/js/video.js

```diff
@@ -433,51 +433,66 @@ videojs.mergeOptions = mergeOptions;
 videojs.bind = Fn.bind;
 
 /**
- * Create a Video.js player plugin
- * Plugins are only initialized when options for the plugin are included
- * in the player options, or the plugin function on the player instance is
- * called.
- * **See the plugin guide in the docs for a more detailed example**
- * ```js
- *     // Make a plugin that alerts when the player plays
- *     videojs.plugin('myPlugin', function(myPluginOptions) {
- *       myPluginOptions = myPluginOptions || {};
+ * Register a Video.js plugin
  *
- *       var player = this;
- *       var alertText = myPluginOptions.text || 'Player is playing!'
+ * @param  {String} name
+ * @param  {Function} plugin
+ *         A sub-class of `Plugin` or an anonymous function for basic plugins.
+ * @return {Function}
+ */
+videojs.registerPlugin = (name, plugin) => Plugin.registerPlugin(name, plugin);
```

Comment:
I had a reason, but I can't remember what it was. So, it couldn't have been too compelling. 😆  I'll change these to be references.